src/pages/docs/headless-cms/extending/content-models-via-code.mdx

```diff
@@ -29,98 +29,130 @@ The most straightforward way to define content models and content model groups w
+    id: "ecommerce",
+    name: "E-Commerce",
+    slug: "e-commerce",
+    icon: "fas/shopping-cart"
   }),
 
   // Defines a new "Product" content model.
   new CmsModelPlugin({
-    name: 'Product',
-    modelId: 'product',
+    name: "Product",
+    modelId: "product",
     group: {
-      id: 'ecommerce',
-      name: 'E-Commerce',
+      id: "ecommerce",
+      name: "E-Commerce"
     },
     fields: [
       {
-        id: 'name',
-        fieldId: 'name',
-        type: 'text',
-        label: 'Product Name',
-        helpText: 'A short product name',
-        renderer: { name: 'text-input' },
+        id: "productName",
+        fieldId: "productName",
+        type: "text",
+        label: "Product Name",
+        helpText: "A short product name",
+        renderer: { name: "text-input" },
         validation: [
           {
-            name: 'required',
-            message: 'Value is required.',
+            name: "required",
+            message: "Value is required."
           },
         ],
       },
       {
-        id: 'sku',
-        fieldId: 'sku',
-        type: 'text',
-        label: 'SKU',
-        placeholderText: 'SKU = Stock Keeping Unit',
-        renderer: { name: 'text-input' },
+        id: "productSKU",
+        fieldId: "productSKU",
+        type: "text",
+        label: "SKU",
+        placeholderText: "SKU = Stock Keeping Unit",
+        renderer: { name: "text-input" }
       },
       {
-        id: 'price',
-        fieldId: 'price',
-        type: 'number',
-        label: 'Price',
-        renderer: { name: 'text-input' },
-      },
+        id: "productPrice",
+        fieldId: "productPrice",
+        type: "number",
+        label: "Price",
+        renderer: { name: "text-input" }
+      }
     ],
-    layout: [['name'], ['sku', 'price']],
-    titleFieldId: 'name',
+    layout: [["productName"], ["productSKU", "productPrice"]],
+    titleFieldId: "productName"
   }),
 ]
 ```
 
-<Alert type="info">
+#### How to Define Field ID's
+<Alert type="danger">
+When defining content models via the code, the field identifiers are not scoped within a model.
+The id attribute must be unique across all your models that are defined via the code.
+</Alert>
 
-Do note that you will also need to import the newly created [`api/code/headlessCMS/src/plugins/models.ts`](https://github.com/webiny/webiny-examples/blob/master/headless-cms/content-model-plugins/basic/api/code/headlessCMS/src/plugins/models.ts) file and register the created plugins in the [`api/code/headlessCMS/src/index.ts`](https://github.com/webiny/webiny-examples/blob/master/headless-cms/content-model-plugins/basic/api/code/headlessCMS/src/index.ts#L42) entrypoint file.
+As a general rule, our suggestion is to prefix the `id` attribute with the model name.
+For example, if you have a model with an `id` of "product" and that model contains a field with an `id` value of "name".
+Instead of just calling it "name", set the `id` to "productName".
```

Comment:
Maybe put this in an info alert?

Reply:
Thanks, Ben, the release links is updated for `v5.25.0` to `v5.26.0`. 
The  `v5.26.0` was not released at the time of writing this doc 🙂 

For the alert part, we already put the introduction part and content related to it in the alert just above this line. This line just includes the instructions, thank you!